apps/air-discount-scheme/backend/src/environments/environment.ts

```diff
@@ -65,8 +86,8 @@ const prodConfig = {
   },
   baseUrl: process.env.BASE_URL,
   identityServerAuth: {
-    issuer: process.env.IDENTITY_SERVER_DOMAIN
-      ? `https://${process.env.IDENTITY_SERVER_DOMAIN}`
+    issuer: isProd
+      ? getStaticEnv('SI_PUBLIC_IDENTITY_SERVER_ISSUER_DOMAIN')
       : '',
```

Comment:
Same comment as above

Reply:
Without it there will be errors when running it locally.